.github/workflows/ci.yaml

```
@@ -62,8 +62,44 @@
+        run: |
+          docker system prune -af || true
+          docker volume prune -f || true
+
+      - name: Run Go Tests (with retry)
+        env:
+          GOMAXPROCS: 1
+          CI: true
+        run: |
+          attempt=1
+          max_attempts=3
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)